tests/test_calculate_metrics.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import pandas as pd
+from numpy.testing import assert_array_equal
+import numpy as np
+
+# pycharm does not like this wants an extra pipeline.import
+# but it works when running pytest in the root (and thus in the pipeline)
+# this will be fixed when we start moving code to backend package.
+from pipeline.calculate_metrics_file import cubic_spline
+
+
+class TestCalculateMetrics(unittest.TestCase):
+    def test_cubic_spline(self):
```

Comment:
This makes sense, I didn't know about the numpy.testing library. 

As a test to ensure that the cubic_spline method actually runs the scipy.interpolate.CubicSpline, this equality test makes sense. 

Also, looking at the function, [cubic_spline](https://github.com/swb-ief/etl-pipeline/blob/master/pipeline/pipeline/calculate_metrics.py), it looks like the function takes in a pandas series, estimates the Cubic Spline function where the x values are [0,length of series], and then just returns the result of the new cubic spline function evaluated at the knot points. Wouldn't this invariably return then values passed into the function? Thought this after reading the third condition on this [reference](https://en.wikiversity.org/wiki/Cubic_Spline_Interpolation). I'll defer to you. 


https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.CubicSpline.html

Reply:
Yeah I wondered that myself. But since this pull request is about the unittest setup I'm going to leave that for now and merge. I will raise an issue though see if we can get some feedback on that on slack.